apps/plugin-browser/src/shared/badge.ts

```diff
@@ -0,0 +1,21 @@
+import browser from "webextension-polyfill";
+
+export const clearBadge = () => {
+  void browser.action.setBadgeText({ text: "" });
+};
+
+export const setLoadingBadge = () => {
+  void browser.action.setBadgeText({ text: "💭" });
+  void browser.action.setBadgeBackgroundColor({ color: "#006DC3" });
```

Comment:
It should be possible to import this value from the design system package:

```suggestion
  void browser.action.setBadgeBackgroundColor({ color: customColors.blue[80] });
```

Reply:
Thank you, fixed in https://github.com/hashintel/hash/commit/f0a263efca4552a51e0c6924f635187caaf961e7